### ISSUE TITLE
Search for new mailinglist

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#529]
Hi Folks,

as Sourceforge seems to revoke all access to our mailinglist I propose to search for a new one. 

```
Your message has been rejected, probably because you are not
subscribed to the mailing list and the list's policy is to prohibit
non-members from posting to it.  If you think that your messages are
being rejected in error, contact the mailing list owner at
geeqie-devel-owner@lists.sourceforge.net.


[-- Anhang #2 --]
[-- Type: message/rfc822, Encoding: 7bit, Size: 4,1K --]

Date: Wed, 25 Oct 2017 19:06:43 +0100
From: Klaus Ethgen <Klaus+geekie@ethgen.de>
To: geeqie-devel@lists.sourceforge.net
Subject: 1.4
```

One that I read about on the dante mailinglist is https://www.freelists.org/signup.html. Any other idea? Best would be if we can use a subdomain of geeqie.org.

**Comment by caclark** [comment]
This GitHub project:
https://github.com/syl20bnr/spacemacs/projects

uses a Label of "Mailing List" to provide a mailinglist feature - in some way or other.
The advantage is that it keeps everything in one place.
I assume there are disadvantages.

As an addendum to this, some years ago I downloaded the full archive of the sourceforge mailing list. I can no longer find it, nor find a way of doing that now.
Does anyone know how to download the archive?

**Comment by mowgli** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Hi,

I was able to export the old mailing list in mbox format. I plan to put
them on the website as an archive.

With that I also deleted old sourceforge list so there is only the link
to the correct page left over there.

I still search for a new mailing list location. In worst case, I will
implement them myself. But I think, the one I would prefer, is to secure
for geeqie. ;-)

Regards
   Klaus
- -- 
Klaus Ethgen                                       http://www.ethgen.ch/
pub  4096R/4E20AF1C 2011-05-16            Klaus Ethgen <Klaus@Ethgen.ch>
Fingerprint: 85D4 CA42 952C 949B 1753  62B3 79D0 B06F 4E20 AF1C
-----BEGIN PGP SIGNATURE-----
Comment: Charset: ISO-8859-1

iQGzBAEBCgAdFiEEMWF28vh4/UMJJLQEpnwKsYAZ9qwFAln+G6UACgkQpnwKsYAZ
9qwNsgv/bKUPZulNohjXf1jVIWEjRXmSCJPFNL5tRnfH1P8eRdb3ivOitrrLtWVd
JEXmdu94mLug+ln08nEQ3pZULtom/eVa2xq6xtQKFInJvlZd2UnVuM5liB2oujyL
vSdMTif8micaxisEMTtXnUm8lPGfMq/wZFfb76S3ne+rwuY/U2ev6MT25OmyFwas
zk4/zdx1OUpCcfLiYnMKViCE+KGY69qwYi9ylicmM9z6zXdangM2HZP65jAXhCgD
LUWolQ0pdfpEz6rI/Ewgnyvy9Wy3TUWmjwcsNFBD+WpK8Ef2RGkYl/6PAIuvXRyv
QpHNwXsVFfhvzUEzofTY8L09tGUVk5GhXammq9nvAcKFk4Qs1wiFkdbKTe1PUgR9
SHVl+EQAKjw34O0EtVEaUqU13VtGsZxOePIJK/eCIoFbHKorKDJqKIMJPHa/ZifG
EtisOdvzE9HHMD0fSfhGNAHhfzZG9jj5PWT6rPkQNc5ANYlwpKTsuxP6ybLqYhSz
a1CkgcCf
=B1kP
-----END PGP SIGNATURE-----
